### DISCUSSION TITLE
建议明确 `sass` 与 `scss` 的区别

### DISCUSSION THREAD

**Discussion by Yue-plus** [vuepress/core#355] [Q&A] [answered]
在 `plugin-palette` 中，使用 SASS 作为 CSS 预处理，却要求提供 `.scss` 文件：

![image](https://user-images.githubusercontent.com/33195150/128591618-358608fd-14ee-4c0a-bcd8-b8c8a285e889.png)
![image](https://user-images.githubusercontent.com/33195150/128591625-5fe00b64-5212-4978-be07-7677eaefd68c.png)

可我不想写花括号和分号（

**Comment by meteorlxy** [comment]
> 可我不想写花括号和分号（

那你就别用 preset ，手动设置各个选项啊

**Reply by Yue-plus** [reply]
只是觉得 `presetOptions` 里应该分一下 `sass` 和 `scss`……

**Reply by Yue-plus** [reply]
不过现在改又会坑到前人2333

**Reply by Mister-Hope** [reply]
我觉得这里的处理是OK的，sass官网的历史版本是有这样一句话的：

![image](https://user-images.githubusercontent.com/33315834/128595476-9c22f47a-4690-47cc-a06d-c63ee2b627cc.png)

而且目前随着 sass3 新推出的 scss 格式也的确是用于最广泛的格式，所以sass代指 scss扩展的格式本身是没有任何问题的。

**Reply by Mister-Hope** [reply]
官方是永远不会放弃缩进式 sass 格式的支持，但是一直在首推 scss，包括现在官网可以找到的描述。

![image](https://user-images.githubusercontent.com/33315834/128595535-6f22f40e-2423-49bf-9303-a8e444ef4075.png)

**Comment by Mister-Hope** [comment, marked as answer]
sass官网的历史版本是有这样一句话的：

![image](https://user-images.githubusercontent.com/33315834/128595476-9c22f47a-4690-47cc-a06d-c63ee2b627cc.png)

而且目前随着 sass3 新推出的 scss 格式也的确是用于最广泛的格式，所以sass代指 scss扩展的格式本身是没有任何问题的。

另外目前 npm 的包名也是 `sass`，所以没有太大必要。
